### PR TITLE
DR dashboard overview, subscriptions, permissions

### PR DESCRIPTION
**Pull request by alex-cit**
related to #436 
Dashboard Redesign - screenshots update:
- https://merginmaps.com/docs/manage/dashboard/
- https://merginmaps.com/docs/manage/subscriptions/
- https://merginmaps.com/docs/manage/permissions/

### CODE REVIEW

File: src/manage/dashboard/mergin-maps-dashboard.jpg
Comment (MarcelGeo) [inline src/manage/dashboard/mergin-maps-dashboard.jpg:1]: @alex-cit - could you please use wider screen for his screenshot? I want to see similar padding around sections as in e.g. src/manage/dashboard/mergin-maps-project-details.jpg . Thanks :)

**Review by MarcelGeo — CHANGES_REQUESTED** [review]

File: src/manage/dashboard/mergin-maps-dashboard-project-history-version.jpg
Comment (MarcelGeo) [inline src/manage/dashboard/mergin-maps-dashboard-project-history-version.jpg:1]: @alex-cit  - could you please use screenshot without "v3" small title in header? We fixed that already - it's not showing after short scroll.

**Review by MarcelGeo — CHANGES_REQUESTED** [review]

**Comment by alex-cit** [comment]
Fixed, thanks for the review!

**Review by MarcelGeo — APPROVED** [review]

File: src/manage/dashboard/index.md [hunk 1]
```diff
@@ -1,71 +1,110 @@
 # Home Page and Dashboard
 [[toc]]
 
-When you log in to <AppDomainNameLink desc="Mergin Maps"/>, you will be presented with a dashboard for quick access to your recent active projects. Here, you can also manage your projects, [workspaces](../workspaces/), profile and [subscriptions](../subscriptions/).
+When you log in to <DashboardLink />, you will be presented with a quick overview of requests and recent active projects in your current workspace. Here, you can also manage your projects, [workspaces](../workspaces/), profile and [subscriptions](../subscriptions/).
 
-![Mergin Maps web dashboard](./web-dashboard-panel.jpg "Mergin Maps web dashboard")
+![Mergin Maps dashboard](./mergin-maps-dashboard.jpg "Mergin Maps dashboard")
 
 Current workspace and your username are displayed in the upper right corner of the screen.
```
Comment (raherin) [inline src/manage/dashboard/index.md:8]: Edit text:
The current workspace and your username are displayed in the upper right corner of the screen.

File: src/manage/dashboard/index.md [hunk 2]
```diff
@@ -1,71 +1,110 @@
...
 **Projects** tab in the left panel displays the list overview of the projects that are available to you in this workspace.
-![Mergin Maps web dashboard project tab](./web-projects-tab.jpg "Mergin Maps web dashboard project tab")
+![Mergin Maps dashboard projects tab](./mergin-maps-dashboard-projects.jpg "Mergin Maps dashboard projects tab")
 
-- click on **+ Create** to [create a new project](../create-project/#create-a-project-through-merginmaps-com), 
-- use **Find Public Projects** to browse public projects
-- select a project to see its details
+Here, you can:
+- use the **Create project** button to [create a new project](../create-project/#create-a-project-through-merginmaps-com)
+- search project by name
```
Comment (raherin) [inline src/manage/dashboard/index.md:18]: Edit text:
- search for a project by name

File: src/manage/dashboard/index.md [hunk 3]
```diff
@@ -1,71 +1,110 @@
...
-**Web Map** feature is available for [Team subscription](https://merginmaps.com/pricing). It displays your <MainPlatformName /> project directly in <AppDomainNameLink />.
+### Project files
+In **Files**, you can see the list of all files associated with the project. 
 
-The content is updated when the project is synchronised. If you see the message "Map config does not exist, please try to update the project", sync the project and the web map should be generated.
+Click on a file to see when it was modified, its size, to download or delete it.
+![Detail of a file in Mergin Maps project](./mergin-maps-dashboard-file-detail.jpg "Detail of a file in Mergin Maps project")
+
+### Webmap
+**Webmap** feature is available for [Team subscription](https://merginmaps.com/pricing).
```
Comment (raherin) [inline src/manage/dashboard/index.md:34]: Edit text:
The **Webmap** feature is available for `[Team subscription](https://merginmaps.com/pricing).`

File: src/manage/dashboard/index.md [hunk 4]
```diff
@@ -1,71 +1,110 @@
...
+
+### Webmap
+**Webmap** feature is available for [Team subscription](https://merginmaps.com/pricing). 
+
+It displays your <MainPlatformName /> project directly in the <DashboardShortLink />. Here, you can see your spatial data without the need to open QGIS or the <MobileAppNameShort />.
+
+![Mergin Maps webmap displays the project content](./mergin-maps-web-map.jpg "Mergin Maps webmap displays the project content")
+
+### History
+**History** contains the overview of available project versions that can be also downloaded to your computer.
```
Comment (raherin) [inline src/manage/dashboard/index.md:41]: Edit text:
The **History** tab contains an overview of available project versions that can be also downloaded to your computer.

File: src/manage/dashboard/index.md [hunk 5]
```diff
@@ -1,71 +1,110 @@
...
 
-## Subscription
-The **Subscription** tab contains information about your [subscription plan](../subscriptions/).
+Here, you can find the current usage of storage, number of users (members and guests) and number of projects in your workspace.
 
-Here, you can find the current usage of storage, number of projects and number of users (members and guests) in your workspace.
+![Mergin Maps Subscriptions details](./mergin-maps-dashboard-subscriptions.jpg "Mergin Maps Subscriptions details")
 
-![Mergin Maps Subscriptions details](../subscriptions/subscriptions.jpg "Mergin Maps Subscriptions details")
+Click on **Manage subscription** to visit the subscription management portal where you can update the subscription plan, billing information, see invoices and upcoming payments.
```
Comment (raherin) [inline src/manage/dashboard/index.md:78]: Edit text:
Click on **Manage subscription** to visit the subscription management portal where you can update the subscription plan, billing information, and see invoices and upcoming payments.

**Review by raherin — CHANGES_REQUESTED** [review]

**Pull request merged**